leo/core/leoAtFile.py

```diff
@@ -2540,27 +2596,25 @@ def precheck(self, fileName: str, root: Position) -> bool:  # pragma: no cover
         Return True if so.  Return False *and* issue a warning otherwise.
         """
         at = self
-        #
+        if g.unitTesting:
+            return True
         # #1450: First, check that the directory exists.
         theDir = g.os_path_dirname(fileName)
         if theDir and not g.os_path_exists(theDir):
             at.error(f"Directory not found:\n{theDir}")
             return False
-        #
         # Now check the file.
         if not at.shouldPromptForDangerousWrite(fileName, root):
             # Fix bug 889175: Remember the full fileName.
             at.rememberReadPath(fileName, root)
             return True
-        #
         # Prompt if the write would overwrite the existing file.
         ok = self.promptForDangerousWrite(fileName)
         if ok:
             # Fix bug 889175: Remember the full fileName.
```

Comment:
same as 5 lines above. "bug fix" to "#".

Reply:
Rev b741add converts two old bug references to the `# #<number>` convetion.

Reply:
You changed those at lines 3075 and 3119, good catch I didnt see those. But those I had marked were at lines 2608 and 2614 (Fix bug 889175: Remember the full fileName.)

Reply:
Let's not worry about the "old-style" comments :-) Either way is clear enough.